### PR TITLE
PERF-5427 cleanup

### PR DESCRIPTION
**Pull request by hilldani**
**Jira Ticket:** PERF-5427

### Whats Changed

I'm removing unused stuff from this workload. It currently has 30 phases but only need 7. The rest are empty. It also has templates for things it doesn't use. Also the test only lasts 5 seconds currently so I'm increasing the runtime to reduce noise.

### Patch Testing Results
[updated patch](https://spruce.mongodb.com/version/6633ca6c4ac0e2000709244f/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)

### CODE REVIEW

**Review by jimoleary — CHANGES_REQUESTED** [review]
Both patches are broken,

File: src/workloads/query/PercentilesAgg.yml [hunk 1]
```diff
@@ -83,81 +91,11 @@ ActorTemplates:
...
-      PhaseConfig:
-        Repeat: 1
-        Collection: *coll
-        Operations:
-        - OperationName: drop
-
 - ActorFromTemplate:
     TemplateName: CreateDataset
     TemplateParameters:
-      OnlyActiveInPhase: 0
```
[the remark this answers is not in the document]
Comment (hilldani) [reply src/workloads/query/PercentilesAgg.yml:160]: oh good catch

File: src/workloads/query/PercentilesAgg.yml [hunk 2]
```diff
@@ -9,6 +9,14 @@ Description: |
   distributions, and therefore scoped down this test to create a normal distribution and a dataset
   created from 5 different distributions.
 
+# 1 - create normal distribution collection
```
[the remark this answers is not in the document]
Comment (hilldani) [reply src/workloads/query/PercentilesAgg.yml:13]: added

File: src/workloads/query/PercentilesAgg.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (hilldani) [reply src/workloads/query/PercentilesAgg.yml]: forgot phases start at 0 not 1 😄

**Comment by hilldani** [comment]
I realized I linked to the wrong patches. I updated it to a more recent test

**Comment by hilldani** [comment]
it actually didn't end up changing the metrics very much. Mostly made profiling better because we get a flamegraph of more than 5 seconds and not 25 empty phases

**Pull request merged**